packages/admin/admin/src/actionLog/diffViewer/DiffViewer.sc.ts

```diff
@@ -0,0 +1,109 @@
+import { css } from "@emotion/react";
+import { Box, type Theme } from "@mui/material";
+import type { ReactDiffViewerStylesOverride } from "react-diff-viewer-continued/lib/cjs/src/styles";
+
+import { createComponentSlot } from "../../helpers/createComponentSlot";
+
+export type DiffViewerClassKey = "root";
+
+export const Root = createComponentSlot(Box)<DiffViewerClassKey>({
+    componentName: "DiffViewer",
+    slotName: "root",
+})(
+    (props: { theme: Theme }) => css`
+        div[role="banner"] {
+            display: none;
+        }
+
+        table {
+            tr:first-of-type {
+                td {
+                    padding: 0;
+                }
+            }
+
+            td:nth-of-type(2) {
+                border-right: 1px solid ${props.theme.palette.grey["200"]};
+            }
+
+            th:first-of-type {
+                border-right: 1px solid ${props.theme.palette.grey["200"]};
+            }
+        }
+    `,
+);
+
+export const createStyles = (theme: Theme): ReactDiffViewerStylesOverride => {
+    return {
+        codeFoldExpandButton: {
+            fontSize: "14px",
+            fontWeight: 600,
+            textDecoration: "underline",
+        },
+        diffAdded: {
+            "& pre": {
+                backgroundColor: "#bdf1d7",
```

Comment:
need to check with @michaelkodim how we are doing with the hard coded color's here.

Reply:
using theme colors `theme.palette.success.main`, and `theme.palette.error.main` with postfixed opacity.